convert2rhel/unit_tests/main_test.py

```diff
@@ -25,17 +25,8 @@
 except ImportError:
     from ordereddict import OrderedDict
 
-from convert2rhel import (  # Imports unit_tests/__init__.py
-    cert,
-    checks,
-    main,
-    pkghandler,
-    redhatrelease,
-    repo,
-    subscription,
-    unit_tests,
-    utils,
-)
+from convert2rhel import main  # Imports unit_tests/__init__.py
+from convert2rhel import cert, checks, pkghandler, redhatrelease, repo, subscription, unit_tests, utils
```

Comment:
Oh I think this comment got broken with the initial black refactor. Comment pointed to `unit_tests` initially

Did `main` import here only split from the rest due to the comment? If you remove the comment does the formatting change?

Reply:
yes, that was due to comment